flutter_mobx/example/lib/todos/todos.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_mobx/flutter_mobx.dart';
+import 'package:mobx/mobx.dart';
+part 'todos.g.dart';
+
+class Todo = TodoBase with _$Todo;
+
+abstract class TodoBase implements Store {
+  TodoBase(this.description);
+
+  @observable
+  String description = '';
+
+  @observable
+  bool done = false;
+}
+
+enum VisibilityFilter { all, pending, completed }
+
+class TodoList = TodoListBase with _$TodoList;
+
+abstract class TodoListBase implements Store {
+  @observable
+  ObservableList<Todo> todos = ObservableList<Todo>();
+
+  @observable
+  VisibilityFilter filter = VisibilityFilter.all;
+
+  @computed
+  ObservableList<Todo> get pendingTodos =>
+      ObservableList.of(todos.where((todo) => todo.done != true));
+
+  @computed
+  ObservableList<Todo> get completedTodos =>
+      ObservableList.of(todos.where((todo) => todo.done == true));
+
+  @computed
+  bool get hasCompletedTodos => completedTodos.isNotEmpty;
```

Comment:
Getters that just read a simple property probably don't need to be `@computed`, just adds extra overhead with no benefit.

Reply:
Agree but this also adds the vocabulary for the domain. Rather than doing `completedTodos.isNotEmpty` every single time, there is a conceptual vocabulary that you can use which is equally reactive. Also want to highlight the fact that you can have computed properties which depend on other computeds